stdlib/contrib/qxip/logql/logql.flux

```diff
@@ -0,0 +1,87 @@
+// Package logql provides functions meant to simplify common logql queries.
+//
+// The primary function in this package is `logql.query_range()`
```

Comment:
It might be worth noting in the description here that all values are returned as string values (using `raw` mode in `csv.from`)